src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1871,7 +1871,7 @@ private void ShowOpenDialogAndOpenResult(object parameter)
 
         private bool CanShowOpenDialogAndOpenResultCommand(object parameter)
         {
-            return HomeSpace.RunSettings.RunEnabled;
+            return HomeSpace.RunSettings.RunEnabled && !HomeSpace.GraphRunInProgress;
```

Comment:
I'm assuming these changes are not really related to this PR but are good anyway to avoid edge cases.

Reply:
These changes are needed because of the refactor I did regarding `RunEnabled`
`RunEnabled` is not set by Dynamo during a graph execution anymore. We introduced the new internal flag `GraphRunInProgress` (previously called `taskExecuting`)

Reply:
The state of things before this PR:
If someone sets the `RunEnabled` flag to false, the Graph Run related UI is disabled and Graph executions are disabled (at least WorkspaceModel::RequestRun is disabled)

Checking for both `HomeSpace.RunSettings.RunEnabled && !HomeSpace.GraphRunInProgress` keeps the same functionality